screen_brightness_control/__main__.py

```diff
@@ -12,8 +13,8 @@ def get_monitors(args):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(prog='screen_brightness_control')
     parser.add_argument('-d', '--display', help='the display to be used')
-    parser.add_argument('-s', '--set', type=str,
-                        help='set the brightness to value between 0 to 100 or in increment as string "+5" or "-5"',
+    parser.add_argument('-s', '--set', type=Union[int, str],
```

Comment:
Apologies, I think one of my comments was unclear. The docstring I wanted you to change was on line 474, changing that to say `value (int or str):` instead of just string. I'm not sure the argparse library supports union types, so just leaving this one as a string is probably fine.